### PR TITLE
[v1.x] Onnx export support for ROIAlign

### PR DESCRIPTION
**Pull request by Zha0q1**

### CODE REVIEW

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 1]
```diff
@@ -3555,3 +3555,42 @@ def convert_broadcast_like(node, **kwargs):
...
+    from onnx import TensorProto
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    pooled_size = convert_string_to_list(str(attrs.get('pooled_size')))
+    spatial_scale = float(attrs.get('spatial_scale'))
+    sample_ratio = int(attrs.get('sample_ratio', '0'))
+    position_sensitive = attrs.get('position_sensitive', 'False')
+    aligned = attrs.get('aligned', 'False')
+
+    '''
```
Comment (josephevans) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: Should these checks be commented out?

**Review by josephevans — APPROVED** [review]

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py (hunk 1 above)
Comment (Zha0q1) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: I realized the issue after I created the pr. I have uncommitted them now : )

**Review by waytrue17 — APPROVED** [review]
LGTM

**Review by josephevans — APPROVED** [review]
LGTM!

**Pull request merged**